Fixes #1621 (#1622)
GEOS-ESM/MAPL: CHANGELOG.md

- Check for duplicate entries in the History.rc file
- Check that a user provided chunking in the History.rc is compatible with the output grid
- If a user request CFIOasync in the History.rc print warning and set to CFIO

> - Make it optional for History to check for pre-existing files before writing

Reviewer: ```suggestion
- Added option allow writing to pre-existing files with History
```
